src/ontology/edits/oeo-physical.omn

```diff
@@ -3441,6 +3443,118 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/755",
         <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00000117
     
     
+Class: OEO_00010085
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A hydro power unit is a power generating unit that uses hydro energy.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/78
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/758",
+        rdfs:label "hydro power unit"@en
+    
+    SubClassOf: 
+        OEO_00000334,
+        OEO_00000503 some OEO_00000441,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00000442
+    
+    
+Class: OEO_00010086
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A hydro powerplant is a powerplant having an aggregate of hydro power generating units as its power generating units parts.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A hydro powerplant is a powerplant having an aggregate of hydro power generating units as its power generating unit parts.",
```

Reply:
I'm not sure if this is grammatically correct now, but the first version seems doubled...